docs/Benchmarking.md

```diff
@@ -1,6 +1,9 @@
-<h1 align="center">Fastify</h1>
+---
+title: Benchmarking
+sidebar_label: Benchmarking
+hide_title: false
```

Comment:
All these settings could be stored in decusaurus?

I think:

- fetch the markdown
- add the meta markdown (I see a pattern here, `hide_title` false and the `title` is the file name without special chars)
- provide the markdown to docusaurus

If it is doable, is it not better to avoid spreading the docusourus config across repositories and branches?